packages/connectivity/README.md

```diff
@@ -41,7 +41,7 @@ If you would like to contribute to the SAP Cloud SDK, please make yourself famil
 
 <br>
 
-- [SAP Cloud SDK Documentation portal](https://sap.github.io/cloud-sdk/)
+- [SAP Cloud SDK Documentation portal](https://sap.github.io/cloud-sdk/api/latest)
```

Comment:
[req] when I follow the link `https://sap.github.io/cloud-sdk/api/latest` in my browser I am redirected to `https://sap.github.io/cloud-sdk/api/2.9.0/` which gives me a page not found. So something is not working.

Reply:
It works locally. Let me check it.

Reply:
The Problem was the API docs are not under baseURL `sap.github.io/cloud-sdk/` because it is static content. I removed `useBaseUrl()` then just navigate to `/cloud-sdk/api/LATEST-VERSION`.